####### The service to set the robot position

### Service inputs

# The robot pose as a 2D coordinate to be stored.
Point pose

---

### No service outputs
